src/Microsoft.DotNet.Interactive.Formatting/DictionaryExtensions.cs

```diff
@@ -4,7 +4,7 @@
 using System;
 using System.Collections.Generic;
 
-namespace Microsoft.DotNet.Interactive.Formatting
```

Comment:
some tools will complain about namespace not matching folders.

Reply:
We can suppress that or use global namespace declarations for the shared files. We're not getting a warning in our CI so I'm not too concerned.